grails-app/services/kuorum/util/rest/RestKuorumApiService.groovy

```diff
@@ -266,6 +266,7 @@ class RestKuorumApiService {
         ACCOUNT_BULLETINS("/communication/campaign/bulletin/{userId}"),
         ACCOUNT_BULLETIN("/communication/campaign/bulletin/{userId}/{campaignId}"),
         ACCOUNT_BULLETIN_COPY("/communication/campaign/bulletin/{userId}/{campaignId}/copy"),
+        ACCOUNT_BULLETIN_COPY_AND_SEND("/communication/campaign/bulletin/{userId}/{campaignId}/copy/send/{contactId}"),
```

Comment:
Again, Could you think in a better name?

Reply:
Changed to:

.../{campaignId}/send/{contactId}